packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/repositories/search_country_code_repository.dart

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https,//www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https,//opensource.org/licenses/MIT.
+
+import '../models/country_code.dart';
+
+class SearchCountryCodeRepository<T> {
+  Future<List<T>> get searchList => Future.delayed(
+        const Duration(seconds: 1),
+        () => _countriesList
+            .map((countryMap) => CountryCodeModel(
+                  name: countryMap.$1,
+                  code: countryMap.$2,
+                ) as T)
+            .toList(),
+      );
+
+  /// region Mock data
+
+  final List<(String, String)> _countriesList = [
```

Comment:
I think here we need to create a new API endpoint for getting the country list

Reply:
Added new endpoint `api/country-codes`.